pom.xml

```diff
@@ -186,11 +186,6 @@
                                     <id>validateAIF-kotlin</id>
                                 </program>
 
-                                <program>
-                                    <mainClass>edu.isi.gaia.ImagesToAIF</mainClass>
-                                    <id>images2Aif</id>
-                                </program>
-   
```

Comment:
The following now needs to be removed from the README:

> You can run this program by running `target/appassembler/bin/images2Aif`.

Reply:
Although actually, you only removed the TEST, not the SOURCE FILE.  In theory we could still build the images2Aif file and create the executable bin, but just not test it as part of a `mvn test` or `mvn install`.

I guess that's up to you.

Reply:
We need to do a more general cleanout of the Coldstart and Kotlin stuff.  I'm going to leave this to that effort.